### PR TITLE
KZOO-304 Add announcement field to accounts.json

### PR DESCRIPTION
**Pull request by dhuynh-Kazoo**

### CODE REVIEW

**Comment by dhuynh-Kazoo** [comment]
Hi @jamesaimonetti  and @icehess, please help me review this

File: doc/ref/maintenance.md [hunk 1]
```diff
@@ -30,7 +30,6 @@
 | `init_apps/1` | `(AppsPath)` | |
 | `init_apps/2` | `(AppsPath,AppUrl)` | |
 | `migrate/0` |  | |
-| `migrate/1` | `(_AccountDbs)` | |
```
Comment (icehess) [inline doc/ref/maintenance.md]: undo this, this not related to your changes

File: doc/ref/maintenance.md [hunk 2]
```diff
@@ -54,4 +53,3 @@
 | `stop_module/1` | `(Module)` | |
 | `update_schema/1` | `(Schema)` | |
 | `update_schemas/0` |  | |
-| `wait_for_prechecks/0` |  | |
```
Comment (icehess) [inline doc/ref/maintenance.md]: undo this, this not related to your changes

File: priv/api/swagger.json [hunk 3]
```diff
@@ -22641,16 +22645,6 @@
             "required": true,
             "type": "string"
         },
-        "MATCH_LIST_ID": {
```
Comment (icehess) [inline priv/api/swagger.json]: undo this, this not related to your changes

File: priv/api/swagger.json [hunk 4]
```diff
@@ -27223,128 +27217,6 @@
                 }
             }
         },
-        "/accounts/{ACCOUNT_ID}/match_lists": {
```
Comment (icehess) [inline priv/api/swagger.json]: undo this, this not related to your changes

File: priv/api/swagger.json [hunk 5]
```diff
@@ -33125,37 +32997,6 @@
                 }
             }
         },
-        "/accounts/{ACCOUNT_ID}/{ENDPOINT_TYPE}/{ENDPOINT_ID}/pushes": {
```
Comment (icehess) [inline priv/api/swagger.json]: undo this, this not related to your changes

File: priv/oas3/oas3-parameters.yml [hunk 6]
```diff
@@ -368,16 +368,6 @@
     'minLength': 32
     'pattern': '^[0-9a-f]+$'
     'type': string
-'MATCH_LIST_ID':
```
Comment (icehess) [inline priv/oas3/oas3-parameters.yml]: undo this, this not related to your changes

File: priv/oas3/openapi.yml [hunk 7]
```diff
@@ -270,11 +270,6 @@
   '/accounts/{ACCOUNT_ID}/lists/{LIST_ID}/entries/{LIST_ENTRY_ID}/vcard':
     '$ref': >-
       paths/lists.yml#/paths/~1accounts~1{ACCOUNT_ID}~1lists~1{LIST_ID}~1entries~1{LIST_ENTRY_ID}~1vcard
-  '/accounts/{ACCOUNT_ID}/match_lists':
```
Comment (icehess) [inline priv/oas3/openapi.yml]: undo this, this not related to your changes

File: priv/oas3/openapi.yml [hunk 8]
```diff
@@ -686,9 +681,6 @@
     '$ref': 'paths/wordle.yml#/paths/~1accounts~1{ACCOUNT_ID}~1wordle'
   '/accounts/{ACCOUNT_ID}/wordle/{WORD_ID}':
     '$ref': 'paths/wordle.yml#/paths/~1accounts~1{ACCOUNT_ID}~1wordle~1{WORD_ID}'
-  '/accounts/{ACCOUNT_ID}/{ENDPOINT_TYPE}/{ENDPOINT_ID}/pushes':
```
Comment (icehess) [inline priv/oas3/openapi.yml]: undo this, this not related to your changes

**Review by icehess — CHANGES_REQUESTED** [review]

**Pull request merged**